HeinOnline.js

```diff
@@ -0,0 +1,240 @@
+{
+	"translatorID": "3dcbb947-f7e3-4bbd-a4e5-717f3701d624",
+	"label": "HeinOnline",
+	"creator": "Frank Bennett",
+	"target": "https?://heinonline.org/HOL/(?:LuceneSearch|Page|IFLPMetaData)\\?",
```

Comment:
escape periods (`\\.org``). Ad `^` at the beginning. And we're sure they never have `www\\.`?